spec/pom.xml

```diff
@@ -17,74 +17,88 @@
 
-    <properties>
-        <site.output.dir>${project.build.directory}/staging</site.output.dir>
-        <maven.site.skip>true</maven.site.skip>
-        <asciidoctor.maven.plugin.version>2.0.0</asciidoctor.maven.plugin.version>
-        <asciidoctorj.pdf.version>1.5.3</asciidoctorj.pdf.version>
-        <spec.name>jakarta-${project.artifactId}-${project.version}</spec.name>
-        <!-- status: DRAFT, BETA, etc., or blank for final -->
-        <status>DRAFT</status>
-        <maven.build.timestamp.format>MMMM dd, yyyy</maven.build.timestamp.format>
-        <revisiondate>${maven.build.timestamp}</revisiondate>
-    </properties>
+    <name>Jakarta XML Binding Specification</name>
 
     <scm>
         <connection>scm:git:git://github.com/eclipse-ee4j/jaxb-api.git</connection>
         <developerConnection>scm:git:git@github.com:eclipse-ee4j/jaxb-api.git</developerConnection>
         <url>https://github.com/eclipse-ee4j/jaxb-api.git</url>
         <tag>HEAD</tag>
     </scm>
-
     <distributionManagement>
         <site>
             <url>scm:git:git@github.com:eclipse-ee4j/jaxb-api.git</url>
         </site>
     </distributionManagement>
 
+    <properties>
+        <site.output.dir>${project.build.directory}/staging</site.output.dir>
+        <maven.site.skip>true</maven.site.skip>
+        <asciidoctorj.version>2.5.2</asciidoctorj.version>
+        <asciidoctorj.pdf.version>1.6.0</asciidoctorj.pdf.version>
+        <jruby.version>9.3.2.0</jruby.version>
+        <spec.name>jakarta-${project.artifactId}-${project.version}</spec.name>
+        <!-- status: DRAFT, BETA, etc., or blank for final -->
+        <status>DRAFT</status>
+        <maven.build.timestamp.format>MMMM dd, yyyy</maven.build.timestamp.format>
+        <revisiondate>${maven.build.timestamp}</revisiondate>
+    </properties>
+
     <build>
         <defaultGoal>package</defaultGoal>
         <plugins>
+            <!-- Sets minimal Maven version -->
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-enforcer-plugin</artifactId>
-                <version>3.0.0-M3</version>
+                <version>3.0.0</version>
                 <executions>
                     <execution>
-                        <id>enforce-versions</id>
+                        <id>enforce-maven</id>
                         <goals>
                             <goal>enforce</goal>
                         </goals>
                         <configuration>
                             <rules>
-                                <requireJavaVersion>
-                                    <version>(,1.9),[11,)</version>
-                                    <message>You need JDK8 or JDK 11 and newer</message>
-                                </requireJavaVersion>
+                                <requireMavenVersion>
+                                    <version>3.6.0</version>
+                                </requireMavenVersion>
```

Comment:
if you want to change this, then require JDK 11+ as well

Reply:
Is there a reason to require the JDK? This specific module builds an HTML file and a PDF file. The other spec projects don't define a JDK version either for the spec module.

Reply:
just consistency - EE 10 requires JDK 11+ to build. If the spec is part of EE 10, doesn't it imply requirement of JDK 11+?

Reply:
note there were issues building the spec project on JDK 9/10 in the past, so even though java as such should not matter, it actually does

Reply:
> doesn't it imply requirement of JDK 11+?

It's a good question, but I don't think it applied to the tools that generate the documents, as I could be done by e.g. Ruby as well. But I'll add the JDK just to be sure, although here too no other projects have it.

The issues on JDK 9/10 were with the ascii doc plug-ins, which weren't ready for them. They are still not absolutely 100%, but close enough.